docs/contributing/how-to-release.md

```diff
@@ -2,14 +2,19 @@
 
 Creating a new release requires a few things before getting started:
 
-* Linux or OSX with 7z installed.
-* Windows with HTML Help Workshop installed.
-* Doxygen 1.8.8, and GraphViz installed on both machines.
+* Linux or OSX.
+* Windows with HTML Help Workshop, and Inno Setup installed.
+* 7-Zip, Doxygen 1.8.8, and GraphViz installed on both machines.
 
 Unless mentioned otherwise, all steps should be run on Linux or OSX so that the
 repository export used for the release is primarily using LF line endings. Files
 that require CRLF line endings will be converted appropriately.
 
+*Important:* Ensure that 7-Zip, HTML Help Workshop, Doxygen, GraphViz and Inno
+Setup have all been added to your Path in Windows. You can confirm this by
+running `7z`, `hhc`, `iscc`, `doxygen -v`, and `dot -V` in a command prompt.
+Add the missing installed folder locations of any executables to your Path.
```

Comment:
I opted to not include instructions for adding these applications to your Path in Windows. It can vary a bit based on the locations you decided to install them at, and also what version of Windows you're running. But I think anyone making releases should already know how to manage their Path in Windows.